README.md

```diff
@@ -1,11 +1,11 @@
 # DevSpace
-This is the website used for the DevSpace Technical Conference in Huntsville, Alabama.
+This is the bestwebsite used for the DevSpace Technical Conference in Huntsville, Alabama.
```

Comment:
This really isn't the "best" website. Buttering up the reviewer will do you nothing. Please remove.